guides/doc-Upgrading_and_Updating/topics/synchronizing_the_new_repositories.adoc

```diff
@@ -31,6 +31,7 @@ In the {ProjectWebUI}, navigate to *Content* > *Subscriptions*, click *Manage Ma
 . In the {ProjectWebUI}, navigate to *Content* > *Sync Status*.
 . Click the arrow next to the product to view the available repositories.
 . Select the repositories for {ProjectVersion}.
+Note that *Red{nbsp}Hat {project-client-name}* does not have a {ProjectVersion} version; simply choose *Red{nbsp}Hat {project-client-name}*.
```

Comment:
I'd probably reword `; simply`, but your call.

Reply:
Thanks for the suggestion! I reworded it a bit, see if you like this version better.